docs/addons/amp-addon.md

```diff
@@ -54,14 +54,15 @@ const addOn = new blueprints.addons.AmpAddOn({
             value: 'Operations',
     }],
     deploymentMode: DeploymentMode.DEPLOYMENT,
-    namespace: 'default'
+    namespace: 'default',
+    name: 'adot-collector-amp'
 })
 
 const blueprint = blueprints.EksBlueprint.builder()
   .addOns(addOn)
   .build(app, 'my-stack-name');
 ```
-Pattern # 3 : Passing on AMP Remote Write Endpoint URL of an existing AMP workspace to be used to remote write metrics. This pattern does not create an AMP workspace. Deploys an ADOT collector on the namespace specified in `namespace` with `deployment` as the mode to remote write metrics to AMP workspace of the URL passed as input. This pattern ignores any other property values passed if `prometheusRemoteWriteURL` is present.
+Pattern # 3 : Passing on AMP Remote Write Endpoint URL of an existing AMP workspace to be used to remote write metrics. This pattern does not create an AMP workspace. Deploys an ADOT collector on the namespace specified in `namespace` with name in `name` and `deployment` as the mode to remote write metrics to AMP workspace of the URL passed as input. This pattern ignores any other property values passed if `prometheusRemoteWriteURL` is present.
 
 ```typescript
 import 'source-map-support/register';
```

Comment:
at some point we will need to clean up all of the source-map-support in the examples. I am unclear why it started in the first place.

Reply:
Sounds good.